ui/readme.de.md

```diff
@@ -152,10 +146,46 @@ Dieses Icon markiert Such-Anfragen deren Ergebnis bei Kunden sehr häufig dazu f
 ![neutral relevance](https://raw.githubusercontent.com/searchhub/searchhub.github.io/master/ui/img/neutral_relevance.png)  
 Dieses Icon markiert Such-Anfragen deren Ergebnis keinerlei Kunden-Interaktion aufweisen. Dies ist sehr häufig ein Indiz für teilweise irrelevante Ergebnisse.
 
+## Underperforming queries{#underperforming-queries}
```

Comment:
In allen Sprachen != "en" muss man `{...}` hinter der Überschrift angeben, damit der link im source code, der immer die englische headline nutzt, passt. Sollte aber die Deutsche Übersetzung den selben Text nutzten, kann man es weglassen. FYI, muss nicht geändert werden,